extensions/amp-next-page/amp-next-page.md

```diff
@@ -25,134 +25,119 @@ limitations under the License.
+
+## Usage
 
 Given a list of pages, `amp-next-page` tries to load them after the current
-document, providing an infinite-scroll type experience.
+document, providing an infinite-scroll experience.
 
 Pages should be specified in a JSON config loaded from a remote URL listed in
-the element `src`, or inlined as a `script` element child.
+the element `src`, or inlined as a `script` element child (or both).
 
 ```html
 <amp-next-page src="https://example.com/next-page-config.json"></amp-next-page>
 <!-- Or -->
 <amp-next-page>
   <script type="application/json">
-    {
-      "pages": ...
-    }
+    [
+      {
+        "url": ...,
+        "title": ...,
+        "image": ...
+      },
+      ...
+    ]
   </script>
 </amp-next-page>
 ```
 
 If loading the next document is successful it will be appended to the end of
-the current document as a child of the `amp-next-page` component, moving any
-content after it further down the page. Because of this the component should
-usually be placed directly after the unique content of a given page: for
-example, at the end of a news article or recipe, but before the footer or
-other content repeated across articles.
+the current document as a child of the `amp-next-page` component. Because of this the component must be the last child of the body element, for
+example, at the end of a news article or recipe. If needed, any footer content should be embedded inside the `<amp-next-page>` tag and will be displayed once no more article suggestions are available.
 
-The component will render a total maximum of three documents on screen in a
-component will render a maximum of three documents (total) on screen at one single instance. This limit may be changed or removed in the future.
+The component will render an infinite number of articles so long as more unique suggestions are provided (either by the remote server through the `src` attribute or by parsing suggestions from the `<amp-next-page>` inline configurations on subsequently loaded documents through the `deep-parsing` attribute). `<amp-next-page>` automatically handles loading and unloading of documents from memory to retain a smooth experience.
 
-[tip type="important"]
-**Important** [`<amp-analytics>`](../amp-analytics/amp-analytics.md) is [currently unsupported](https://github.com/ampproject/amphtml/issues/15807) on pages users land on through `<amp-next-page>`.
-Tracking page views is supported through [`<amp-pixel>`](../../builtins/amp-pixel.md) or `<amp-analytics>` on the host page.
-[/tip]
+### Footer Recommendation Box
 
-### Analytics triggers
+If the user reaches the end of a page before the next has loaded, or if the
+next page fails to load or if the maximum number of recommendations is reached (as specified by `max-pages`) a box will be displayed with links to the remaining pages. See the Styling section for information on how to customize the default recommendation box.
```

Comment:
Update section to:
```
The <amp-next-page> component renders the footer recommendation box if one of the following situations arise:

- The user reaches the end of the page before the next page had loaded.
- The next page fails to load.
- If the max-pages attribute is specified and the number of specified pages is met.


The recommendation box contains links to the remaining pages. It renders the specified image and title used in the JSON configuration.  
```

Reply:
Question: it says the box specified the remaining pages - how is this possible if the `max-pages` number is met?

Reply:
Essentially if the configuration has 10 recommended articles but `max-pages` is 6, the recommendation box will have the last four that were not loaded/appended